fluentd/fluent.conf

```diff
@@ -112,8 +112,10 @@
   tag system.var.log
   format multiline
   format_firstline /^[A-Z][a-z]{2} [ |0-3]+[0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
-  format1 /^(?<time>[^ ]* [^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
+  format1 /^(?<time>[^ ]*[ ]*[^ ]*[ ]*[^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
```

Comment:
Could you include a comment indicating examples of what you do and don't want to match here? It would help evaluate whether the regex is doing the right thing.

Also \s (space) and \S (non-space) would be easier to read than the bracket notation (for one versed in regex at least). And I wonder if you want + instead of * ?